data/summary/store/store.go

```diff
@@ -271,9 +271,13 @@ func (r *Repo[T, A]) GetMigratableUserIDs(ctx context.Context, page *page.Pagina
 		return nil, errors.New("pagination is missing")
 	}
 
+	// Ensure we do not select items which will be picked up by GetOutdatedUserIDs
+	// 2 minutes buffer to exclude 90s jellyfish buffer
+	exclusionTime := time.Now().UTC().Add(2 * time.Minute)
+
 	selector := bson.M{
 		"type":                 types.GetTypeString[T, A](),
-		"dates.outdatedSince":  nil,
+		"dates.oudatedSince":   bson.M{"$not": bson.M{"$lte": exclusionTime}},
```

Comment:
typo? `oudatedSince` vs. `outdatedSince`

Reply:
yes, thank you.